src/main/java/edu/elka/peakadvisor/CassandraConfig.java

```diff
@@ -0,0 +1,62 @@
+package edu.elka.peakadvisor;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.cassandra.config.CassandraClusterFactoryBean;
+import org.springframework.data.cassandra.config.CassandraSessionFactoryBean;
+import org.springframework.data.cassandra.config.SchemaAction;
+import org.springframework.data.cassandra.convert.CassandraConverter;
+import org.springframework.data.cassandra.convert.MappingCassandraConverter;
+import org.springframework.data.cassandra.core.CassandraOperations;
+import org.springframework.data.cassandra.core.CassandraTemplate;
+import org.springframework.data.cassandra.mapping.BasicCassandraMappingContext;
+import org.springframework.data.cassandra.mapping.CassandraMappingContext;
+import org.springframework.data.cassandra.mapping.SimpleUserTypeResolver;
+
+/**
+ * Created by apiotrowski on 25.05.2017.
+ */
+@Configuration
+public class CassandraConfig {
+
+    @Bean
+    public CassandraClusterFactoryBean cluster() {
+
+        CassandraClusterFactoryBean cluster = new CassandraClusterFactoryBean();
+        cluster.setContactPoints("localhost");
```

Comment:
spoko spoko, jak coś to proces można włączyć w tle './cassandra &' i będzie już śmigało na wieki